cmd/minikube/cmd/image.go

```diff
@@ -310,6 +312,22 @@ var buildImageCmd = &cobra.Command{
 				// Otherwise, assume it's a tar
 			}
 		}
+		if runtime.GOOS == "windows" && strings.Contains(dockerFile, "\\") {
+			// if dockerFile is a DOS path, translate it into UNIX path
+			// but we need to ask for permission from user first
+			out.String("minikube detects that you are using DOS-style path %s. Should minikube convert it to UNIX-style by replacing all \\ to /? [y/n]", dockerFile)
+			var answer string
+			for {
+				fmt.Scan(&answer)
```

Comment:
Many people use minikube in their CI for testing. If someone is running this command in a script it will hang forever since there will be no input.

Either some sort of interaction flag will need to be added so users running via scripts can bypass this, or instead of automating this we can just suggest that the user makes this change themself.